src/program-editors.bib

```diff
@@ -0,0 +1,44 @@
+@misc{mealin2012exploratory,
+  title={An exploratory study of blind software developers},
+  author={Mealin, Sean and Murphy-Hill, Emerson},
+  year={2012},
+  organization={IEEE}
+}
+ 
+@article{armaly2017comparison,
+  title={A comparison of program comprehension strategies by blind and sighted programmers},
+  author={Armaly, Ameer and Rodeghero, Paige and McMillan, Collin},
+  journal={IEEE Transactions on Software Engineering},
+  year={2017},
+  publisher={IEEE}
+}
+ 
+@misc{doustdar_2016, title={Parham Doustdar's Blog}, url={https://www.parhamdoustdar.com/2016/03/27/autobiography-blind-programmer/}, journal={Parham Doustdar's Blog}, author={Doustdar, Parham}, year={2016}, month={Mar}}
```

Comment:
The title of this should be the title of the blog post, not just "Parham Doustdar's Blog"

Reply:
put newlines between items in a bibtex entry